legacy-app/src/app/modules/events/components/event-calendar/event-calendar.component.html

```diff
@@ -1,41 +1,55 @@
-<div class="mt-4 mb-2">
-  Get an overview about the upcoming events. Events without free spots are
-  <span class="opacity-40">in a lighter color</span>. If the registration is not
-  yet open, you can see a 🔒 with the date of the registration.
-</div>
-<div gdAuto gdColumns="repeat(7, 1fr)" class="border-r border-b">
-  <div class="border-t border-l p-1" *ngFor="let weekday of weekdays">
-    <p>{{ weekday }}</p>
+<div
+  gdColumns="repeat(7, calc(100% / 7))"
```

Comment:
```suggestion
  gdColumns="repeat(7, 1fr)"
```
Should be the correct way to do this in css grid